### ISSUE TITLE
A script in this page is causing your browser to run slowly.

### ISSUE THREAD

**Issue by JoshWho** [WWBN/AVideo#6782]
Have anyone ever figured out what keeps causing this. I kinda think it might be the socket thing but not sure.

Every page if left long enough and come back too. this doesnt happen with outer websites just mine.

**Comment by JoshWho** [comment]
Seems like if you keep any of the pages open for a long period it starts doing it. Almost like thee is some script that is lively running and if there is a disconnection it makes the whole page freeze until it reconnects.

**Comment by DanielnetoDotCom** [comment]
yes, it may happen, but I also leave multiple AVideo sites open for days I am not facing this problem.

I suspect it may be some other script that does not belong to AVideo by default
